src/timbot.py

```diff
@@ -0,0 +1,193 @@
+import io
+import os
+import sys
+import time
+import yaml
+import random
+import datetime
+import numpy as np
+import database as db
+import mysql.connector
+from slackclient import SlackClient
+from mysql.connector import errorcode
+
+
+ideal_lunch_time = "15:30"
+friday_index_elem = 4
+saturday_index_elem = 5
+sunday_index_elem = 6
+help_text = '''
+    You can ask me the following:
+    `where lunch`: i tell you where to eat lunch
+    `where lunch no feast`: i tell you where to eat lunch that isn't epicurean feast
+    `when lunch/lunch time`: i tell you when to eat lunch
+    `what lunch/what eat`: i tell you what to eat
+    `show webopoly standings`: show who is captalist king and who is poor
+    `help`: this
+'''
+
+
+# TODO: Change this to handle daylight savings
+def utc_to_est(utc_time):
+    return str((int(utc_time[:2]) - 5) % 24) + ':' + utc_time[-2:]
```

Comment:
Does this fix #64?

Reply:
I didn't touch this function. You may have to look at the commits to see the changes I made.

Reply:
Since the file location changed, the diff shows the entire file as an addition